src/qmapshack/canvas/CCanvas.cpp

```diff
@@ -1120,20 +1120,23 @@ void CCanvas::displayInfo(const QPoint& px)
     QToolTip::hideText();
 }
 
-poi_t CCanvas::findPOICloseBy(const QPoint& px) const
+QList<poi_t> CCanvas::findPOICloseBy(const QPoint& px, QList<QPointF>& posPOIHighlight) const
```

Comment:
Please add some documentation in the header. It's really hard to understand what posPOIHighlight is good for and how it is filled. And how it is different to the returned list of poi_t and why it can't be derived from that list.